docs/source/nlp/api.rst

```diff
@@ -57,6 +57,8 @@ Modules
     :show-inheritance:
     :members:
 
```

Comment:
Do you know why the API for the NLP models aren't showing up?

Reply:
`get_pretrained_lm_model` got renamed and the docs failed to build, this PR fixes that